filters/scheduler/doc.go

```diff
@@ -0,0 +1,39 @@
+// Package scheduler implements filter logic that changes the http
+// request scheduling behavior of the proxy.
+//
+// The proxy has as default an unbounded scheduler that does not limit
+// inflight requests. Goroutines with parsed request data consume
+// memory. The unbounded handler could spike in memory, if you have
+// traffic on a backend that has too big response times. You can check
+// the number of goroutines from skipper metrics, if you have this
+// problem.
+//
+// The scheduler filter package has two implementations of bounded
+// queue, the lifo and lifoGroup filter. Both lifo filters will, use a
+// last in first out queue to handle most requests fast and if skipper
+// is in an overrun mode, it will serve some requests fast and some
+// will timeout. The idea is based on Dropbox bandaid proxy, which is
```

Comment:
This is the third place where the `bandaid` proxy is mentioned. Maybe we can drop it here in the `godoc`